10 Indicator Fixes (alticons)/gamedata/scripts/z_alticons.script

```diff
@@ -0,0 +1,56 @@
+size_adjust = 0.75
+
+function alticons.actor_status.UIIndicators:InitControls()
+	ratio = (device().height / device().width) / (768 / 1024)
+	self.W = 39 * size_adjust * ratio --Only change by amout of margin in the icon textures
+	self.offset = 2 * ratio * size_adjust -- controls the spacing between icons
+	
+	local xml = utils_xml.get_hud_xml()
+	
+	local time_xml = CScriptXmlInit()
+	time_xml:ParseFile("actor_menu.xml")
+	
+	self.dialog = xml:InitStatic("indicators", self)
+	pos = self.dialog:GetWndPos()
+	DART.alticons_x = pos
+	-- DART FIX for centering indicators on ultra widescreens
+	self.dialog:SetWndPos( vector2():set(pos.x + DART.HUD_SAFE_ZONE, pos.y - 8)) --nudgeing it up slightly for reasons. might add an adjustment in game for this as well.
```

Comment:
My only edit, rest is copied from UI Rework. Realizing know i need to compare with `alticons` to take into account what UI Rework changed

Reply:
Cool this is for the status icons, right?

Reply:
Yes, alticons is the drink radiation and food sensors to the bottom right of the bhs